File: Modul 4 Sem 1/PHYTHON/PRAK402-2410817110005-MuhammadRizqanKarim.py
batas = int(input("Masukkan batas maksimal: "))

print("Bilangan ganjil: ", end="")
for i in range(1, batas + 1, 2):
    print(i, end=" ")

print("\nBilangan genap: ", end="")
for i in range(batas if batas % 2 == 0 else batas - 1, 1, -2):
    print(i, end=" ")

File: Modul 4 Sem 1/C/PRAK402-2410817110005-MuhammadRizqanKarim.c
#include <stdio.h>

int main() {
    int batas;
    printf("Masukkan batas maksimal: ");
    scanf("%d", &batas);

    printf("Bilangan ganjil: ");
    for (int i = 1; i <= batas; i += 2) {
        printf("%d ", i);
    }

    printf("\nBilangan genap: ");
    for (int i = (batas % 2 == 0 ? batas : batas - 1); i >= 2; i -= 2) {
        printf("%d ", i);
    }

    return 0;
}